graphql/resolvers.go

```diff
@@ -4104,24 +4093,12 @@ func (r *versionResolver) Status(ctx context.Context, obj *restModel.APIVersion)
 			patchStatuses := []string{*p.Status}
 			for _, cp := range p.ChildPatches {
 				patchStatuses = append(patchStatuses, *cp.Status)
-				// add the child patch tasks to tasks so that we can consider their status
```

Comment:
Can you please explain the motivation behind this change a bit more?

Reply:
We want to fetch version status for up to 100 versions at a time and querying for version tasks makes this expensive. A version is "aborted" when at least one task is "aborted" and none are "failed." These code changes from [EVG-15555](https://github.com/evergreen-ci/evergreen/pull/5479) create an aborted field for versions. The hope is we can examine the status field and aborted field of a version to determine it's status as opposed to looping through tasks. 

I think if the child patch tasks don't set the parent version's aborted field, we will have to update the GetCollectiveStatus  to consider the aborted field of the child patches version. Let me know what you think and if this sounds right!

Reply:
I changed the logic to fetch the version associated with the child patch to whether it has been aborted. Child patches won't set the aborted field on their parent patch.